### DISCUSSION TITLE
Exporting and importing database within the same project

### DISCUSSION THREAD

**Discussion by muditchordia** [LCA-ActivityBrowser/activity-browser#1390] [Q&A] [answered]
Hi

I am trying to export database (with activities) and editing flow quantities in excel and importing them back within the same project. When I do that, the system goes into an infinite loop without any results (Writing datasets to SQLite databse). I rename the database in the excel to avoid any naming conflicts.

I notice that in the terminal window i get a message: Can't find migration biosphere-2-3-categories. I am not sure why did would happen as I am still operating within the same project. 

Any suggestions on how to deal with this?

**Comment by marc-vdm** [comment, marked as answer]
Yes we had an error in a previous version which didn't correctly set some Brightway functionality when you would set up a project with that version. 

The steps to solve are:
1. [Update Activity Browser](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/Installation-Guide#updating-activity-browser)
2. Start Activity Browser (`conda activate ab`  and then `activity-browser`)
3. Go to top left, `Project` > `Migrations` > `Install default migrations`
4. Restart your import.

**Reply by muditchordia** [reply]
Thanks this helped. I also had to remove the columns “clasifications” and “properties” from the excel for it to work as they have "nan" values.
